jmclient/jmclient/wallet_utils.py

```diff
@@ -548,9 +549,13 @@ def get_addr_status(addr_path, utxos, is_new, is_internal):
     else:
         return walletview
 
-def cli_get_wallet_passphrase_check():
-    password = get_password('Enter wallet file encryption passphrase: ')
-    password2 = get_password('Reenter wallet file encryption passphrase: ')
+def cli_get_wallet_passphrase_check(changing_passphrase=False):
+    if changing_passphrase:
+        password = get_password('Enter new wallet file encryption passphrase: ')
+        password2 = get_password('Reenter new wallet file encryption passphrase: ')
+    else:
+        password = get_password('Enter wallet file encryption passphrase: ')
+        password2 = get_password('Reenter wallet file encryption passphrase: ')
```

Comment:
This duplication isn't needed. Just have a string var which is `""` if not `changing_passphrase` else `"new"` and then construct the argument to `get_password` from that.

Reply:
Cleaned up also part of the code that asks for the *old* passphrase. See the new commits.